### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by cbsaikumar**
Please review and let me know the feedback.

### CODE REVIEW

File: src/expressjs/config/strategies/facebook.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const facebookStrategy = () => {
+   return new FacebookStrategy({
+        clientID: config.facebook.clientID,
+        clientSecret: config.facebook.clientSecret,
+        callbackURL: config.facebook.callbackUrl
+      },
+        (accessToken, refreshToken, profile, done) => {
+            return done(null, profile, {
+                'code': 200,
```
Comment (abruzgin) [inline src/expressjs/config/strategies/facebook.js]: No need to surround keys in brackets

File: src/expressjs/config/strategies/local.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import * as config from '../config';
+
+export const localStrategy = () => {
+   return new LocalStrategy((username, password, cb) => {
+       const {username: configUsername, password: configPassword} = config.user;
+       if(username !== configUsername || password !== configPassword){
+            return cb(null, false);
+        } else{
+            return cb(null, config.user);
+            }
```
Comment (abruzgin) [inline src/expressjs/config/strategies/local.js]: seems like unused curly brace

File: src/expressjs/config/strategies/facebook.js (hunk 1 above)
Comment (cbsaikumar) [reply src/expressjs/config/strategies/facebook.js]: Thanks updated the code

File: src/expressjs/config/strategies/local.js (hunk 2 above)
Comment (cbsaikumar) [reply src/expressjs/config/strategies/local.js]: Thanks updated the code

**Review by abruzgin — APPROVED** [review]

**Pull request merged**